include/Dialects/VectorExt/VectorExtOps.td

```diff
@@ -148,8 +148,9 @@ def VectorExt_WarpSingleLaneOp : VectorExt_Op<"warp_execute_on_lane_0",
   let skipDefaultBuilders = 1;
   let builders = [
     OpBuilder<(ins "Value":$laneid)>,
-    // TODO: Add arguments support in the builder.
-    OpBuilder<(ins "TypeRange":$resultTypes, "Value":$laneid)>
+    OpBuilder<(ins "TypeRange":$resultTypes, "Value":$laneid)>,
+    OpBuilder<(ins "TypeRange":$resultTypes, "Value":$laneid,
+                  "ValueRange":$operands, "TypeRange":$argTypes)>
```

Comment:
nit: indent

Reply:
nit: I would call them `$args` instead of `$operands`

Reply:
I would rename `argTypes` to `blockArgTypes` and add a comment that this cannot be inferred from `args` and is needed to distinguish between broadcasting/gathering from lanes (are these the right terms?) semantics.

Reply:
done.